docker/Dockerfile_db

```diff
@@ -0,0 +1,9 @@
+FROM candidate-registry.fedoraproject.org/postgresql
+
+USER root
+
+# Adds the semver extension on top of the official image
+RUN dnf install -y wget pg-semver && \
+    dnf clean all
+
+USER postgres
```

Comment:
The final USER declaration should use numeric values. 

https://docs.openshift.com/container-platform/3.7/creating_images/guidelines.html#openshift-specific-guidelines